Scripts/script-ExportToCSV.yml

```diff
@@ -32,7 +32,13 @@ script: |-
                   line += ',';
               }
 
-              line += array[i][header];
+              var value = array[i][header] + '';
+              value = value.trim();
+              if (value && value.indexOf(',') > -1) {
+                  line += '"' + value + '"';
```

Comment:
There are more csv escaping rules you need to follow:
https://tools.ietf.org/html/rfc4180